pom.xml

```diff
@@ -212,6 +212,7 @@
 
         <maven.plugin.build.helper.version>3.2.0</maven.plugin.build.helper.version>
         <maven.plugin.download.version>1.6.6</maven.plugin.download.version>
+        <maven.plugin.enforcer.mojo.extra.rules.version>1.6.1</maven.plugin.enforcer.mojo.extra.rules.version>
```

Comment:
maven.plugin.enforcer.extra.rules.version

Reply:
`mojo` should be kept in the prop name, as `org.codehaus.mojo:extra-enforcer-rules` is third-party rules provider for mavn enforcer plugin.

Reply:
Under the premise of being identifiable, we prefer the short name.

Reply:
What about `maven.plugin.enforcer.mojo.rules.version` ?